### PR TITLE
Component next-steps

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Extracted `next-steps` into a new component.

### CODE REVIEW

File: app/scripts/directives/create/nextSteps.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  ctrl.allTasksSuccessful = function(tasks) {
+    return !pendingTasks(tasks).length && !erroredTasks(tasks).length;
+  };
+
+  ctrl.erroredTasks = erroredTasks;
+  ctrl.pendingTasks = pendingTasks;
+
+  ctrl.toggleParamsTable = function() {
+    console.log('did it toggle?');
```
Comment (jwforres) [inline app/scripts/directives/create/nextSteps.js]: remove

**Comment by jwforres** [comment]
just the one nit, you double checked next steps coming in from both a builder image flow and a template processing flow and both seemed ok?

File: app/scripts/directives/create/nextSteps.js (hunk 1 above)
Comment (benjaminapetersen) [inline app/scripts/directives/create/nextSteps.js]: ah, oop. will get that.

**Comment by benjaminapetersen** [comment]
updated & rebased

**Comment by benjaminapetersen** [comment]
@jwforres yup, did both flows, env table still fills out, seems to be in good shape!

File: app/index.html [hunk 2]
```diff
@@ -338,6 +338,7 @@ <h2 class="text-center" id="temporary-loading-message" style="display: none;">Lo
         <script src="scripts/directives/buildStatus.js"></script>
         <script src="scripts/directives/routeServiceBarChart.js"></script>
         <script src="scripts/directives/bindService.js"></script>
+		<script src="scripts/directives/create/nextSteps.js"></script>
```
Comment (jwforres) [inline app/index.html]: something is off with the indent here

**Comment by benjaminapetersen** [comment]
@jwforres fixed the nits & rebased

**Comment by jwforres** [comment]
@benjaminapetersen travis isn't happy, think you need to run bower install

**Comment by benjaminapetersen** [comment]
ah, bother. got it.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**